pkg/datastore/example_multiinstance_test.go

```diff
@@ -38,54 +41,55 @@ func ExampleDataStore_multiInstance() {
 	DevInstanceCtx := instancer.WithInstanceId(ServiceAdminCtx, "Dev")
 	ProdInstanceCtx := instancer.WithInstanceId(ServiceAdminCtx, "Prod")
 
+	// Initializes the Datastore using the metadata authorizer and connection details obtained from the ENV variables.
 	ds, _ := datastore.FromEnv(datastore.GetCompLogger(), mdAuthorizer, instancer)
```

Comment:
I propose we don't ignore the second output parameter (error) and instead print a message at Fatal level, to show the users a good example - if `FromEnv` method fails, then there is no point of continuing